src/e2e-app/button/button-demo.html

```diff
@@ -0,0 +1,19 @@
+<div class="demo-button">
```

Comment:
Can we call these files something other than `button-demo`? I'd generally like the examples to be more like `simple-buttons`, `tabs-inside-dialog`, etc.